File: kafka-connector-frontend/article-analyzer/src/api/kafkaTest.js
const { Kafka } = require('kafkajs')

const kafka = new Kafka({
    clientId: 'article-analyzer',
    brokers: ['10.123.252.211:9092', '10.123.252.212:9092', '10.123.252.213:9092']
  })

const AnalyzingTopic = "grammefars_test"
const producer = kafka.producer()

const helloKafka = async() => { 
await producer.connect()
    .catch(e => {
        console.log(e)
    })
    await producer.send({
        topic: AnalyzingTopic,
        messages: [
            {value: "Hello world"}
        ],
    })
    .catch(e => { 
        console.log(e)
    })
}
    

File: article-analyzer/Controllers/KafkaTest.cs
using Confluent.Kafka;

namespace article_analyzer.Controllers
{
    public class KafkaTest 
    {
        public KafkaTest() {
            var config = new ProducerConfig { 
                ClientId = "article-analyzer-frontend",
                BootstrapServers = "10.123.252.211:9092, 10.123.252.212:9092, 10.123.252.213:9092",
            };

            using (var producer = new ProducerBuilder<Null, string>(config).Build()) {
                producer.Produce("grammefars_test", new Message<Null, string>{Value ="Hello world from dotnet"});
            }
        }
    }
}
